.circleci/config.yml

```diff
@@ -47,6 +47,7 @@ jobs:
             "$(which node)" setup.js up -nc 3
             cd /home/circleci/mad-dog/
             docker ps
+            set -e
```

Comment:
sry to jump on this so quickly but we should fail if local setup fails too

Reply:
i can move this earlier to fail if anything fails